### PR TITLE
feature : Docker todo ui update

### PR DESCRIPTION
**Pull request by lebe24**
Hi @nunocoracao 
Updated the docker todo ui to make it more modernised and not boring. This is a pull request of #23.

### CODE REVIEW

**Comment by nunocoracao** [comment]
Hello @lebe24, thanks for the PR. I'll reach out to the right team internally so that they can review - I am not contributing to this repo atm.

File: app/views/todos.ejs [hunk 1]
```diff
@@ -12,20 +12,32 @@
...
-            <div class="row">
-                <div class="col-md-8 m-auto pt-4">
+    <div class="container m-5 p-2 rounded mx-auto bg-light shadow">
+        
+        <!-- App title section -->
+        <div class="row m-1 p-4">
+            <div class="col">
+                <div class="p-1 h1 text-primary text-center mx-auto display-inline-block">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 640 512" width="50" height="50">
+                        <!-- !Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free Copyright 2024 Fonticons, Inc. -->
```
Comment (sfavello) [inline app/views/todos.ejs:22]: Hello, thank you so much for contributing! We appreciate it. Would it be possible to take out the logo and change the `todo's` to `Todo App`

**Review by sfavello — CHANGES_REQUESTED** [review]
Everything looks great just a small change and we're good to go. Thank you again for contributing. This is a lovely upgrade.

File: app/views/todos.ejs (hunk 1 above)
Comment (lebe24) [reply app/views/todos.ejs:22]: HI @sfavello  sure i can make the necessary changes, I added the docker logo for brand sake but I can make the change and also thanks for reviewing the code.

**Review by sfavello — APPROVED** [review]
LGTM! 🚀 Thank you again for the contribution. We greatly appreciate it!

**Comment by lebe24** [comment]
Awesome @sfavello thanks for the review looking forward to contributing to more repo's in the future

**Review by dreadwitdastacc-IFA — APPROVED** [review]

**Pull request merged**